app_backend/balance.py

```diff
@@ -0,0 +1,55 @@
+from user import User
+
+
+class BalanceTracker(User):
+    def __init__(self, name, dooley_dollars, meal_swipes):
+        super().__init__(name, dooley_dollars, meal_swipes)
+        """
+
+        name: str
+            The name of the user
+
+        dooley_dollars: float
+            The current balance of dooley dollars
+
+        meal_swipes: int
+            The current amount of remaining meal swipes
+
+        """
+
+    # ------------ Decrementing Balances (for spending) -------------
+
+    def dooley_sub(self, spent):
+        """
+        spent: The amount of dooley dollars spent
+        return: The updated Balance Tracker object
+        """
+        self.dooley_dollars = self.dooley_dollars - spent
```

Comment:
what if it reaches 0 or goes under 0? should we add a check for that? and same for the mealswipe_sub